src/main/kotlin/com/openlattice/hazelcast/processors/organizations/UpdateOrganizationExternalDatabaseTableEntryProcessor.kt

```diff
@@ -0,0 +1,34 @@
+package com.openlattice.hazelcast.processors.organizations
+
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.edm.requests.MetadataUpdate
+import com.openlattice.organization.OrganizationExternalDatabaseTable
+import java.util.*
+
+data class UpdateOrganizationExternalDatabaseTableEntryProcessor(val update: MetadataUpdate):
+        AbstractRhizomeEntryProcessor<UUID, OrganizationExternalDatabaseTable, OrganizationExternalDatabaseTable?>() {
+
+    override fun process(entry: MutableMap.MutableEntry<UUID, OrganizationExternalDatabaseTable>): OrganizationExternalDatabaseTable? {
```

Comment:
I think here also, the return value shouldn't be nullable if the entry value is not nullable